.travis.yml

```diff
@@ -11,4 +11,5 @@ install:
   - go get -u github.com/golang/dep/...
   - dep ensure
 script:
-  - make
\ No newline at end of file
+  - make
+  - make test
```

Comment:
nit: not sure about the last symbol, add a blank line by the end?

Reply:
+1, files normally should end with a newline.